lerna.json

```diff
@@ -1,5 +1,9 @@
 {
-  "npmClient": "yarn",
-  "useWorkspaces": true,
-  "version": "2.0.0-alpha.7"
+    "npmClient": "yarn",
+    "useWorkspaces": true,
+    "version": "2.0.0-alpha.8",
+    "packages": [
+        "./packages/lib/*",
+        "./packages/ui/react-components"
+    ]
```

Comment:
Lerna should use yarn's "workspaces" entry in package.json. Are they needed here as well?

Reply:
I don't think they are - might be a legacy of me trying to get things to work; will try removing it and checking if it still works